main.go

```diff
@@ -267,8 +269,14 @@ func cgroupDirPath() (string, error) {
 	for scanner := bufio.NewScanner(cgroupInfo); scanner.Scan(); {
 		line := scanner.Text()
 		if devicesCgroup.MatchString(line) {
-			return devicesCgroup.FindStringSubmatch(line)[1], nil
+			return podCgroupDirPath(devicesCgroup.FindStringSubmatch(line)[1]), nil
 		}
 	}
 	return "", errors.New("can't find out cgroup directory")
 }
+
+func podCgroupDirPath(containerCgroupDirPath string) string {
+	split := strings.Split(containerCgroupDirPath, string(filepath.Separator))
+	split[len(split)-1] = "*" // any container match
```

Comment:
Why do we need to replace here?

Reply:
For kubernetes we have such cgroup path for the container:
```
.../pod-id/container-id
```
so if we want select all containers in pod, it should be:
```
.../pod-id/*
```